eur_nkg/eur_nkg_README.txt

```diff
@@ -18,6 +18,18 @@ examples of use.
 
 * eur_nkg_nkgrf03vel_realigned.tif
 
+### Nordic+Baltic countries: 2017 Deformation model
+
+*Source*: [The Nordic Geodetic Commission](https://http://www.nordicgeodeticcommission.com/)
```

Comment:
For odd formatting reason, you need to add two spaces at the end of that line and the following one. See how it renders currently: https://github.com/OSGeo/PROJ-data/tree/1345aa3b88616bb8beb63b5cf73cc0506dc4befa/eur_nkg

Reply:
Right, yes. Should be fixed now.